docs/Components/Aerodynamic/RectangularHorizontalStabilizer.md

```diff
@@ -0,0 +1,19 @@
+## RectangularHorizontalStabilizer
+This class creates a rectangular horizontal stabilizer
```

Comment:
It would be good to better describe it, like the needed arguments for the instantiation.